django_tables2/columns/base.py

```diff
@@ -215,6 +215,9 @@ class Column(object):
              - If a `dict` is passed, it's passed on to ``~django.urls.reverse``.
              - If a `tuple` is passed, it must be either a (viewname, args) or (viewname, kwargs)
                tuple, which is also passed to ``~django.urls.reverse``.
+        initial_sort_descending (bool): If `True`,
+            column.order_by_alias.next will return descending ordering if not ordered.
```

Comment:
This is currently explained in terms of internals. This makes sense while implementing it, but I'd rather see it explained in end user problems/solutions. Can you change the wording?